Yoast/javascript
# P2 694 create schema blocks redux store

## Summary


This PR can be summarized in the following changelog entry:

* [@yoast/schema-blocks] creates a redux store for schema blocks .

## Relevant technical choices:

* moved reducer, actions and selectors from free ( see https://github.com/Yoast/wordpress-seo/pull/16825 ) to the schema blocks monorepo package.
* added Typescript type hints.
* organized the module structure.

## Test instructions

This PR can be tested by following these steps:

* add any yoast block; this will trigger validation, which should be stored in redux.
* use the redux DevTools to analyze the yoast/schema-blocks store, the validation results should be there.
![image](https://user-images.githubusercontent.com/3604126/112845023-a9a2ff00-90a4-11eb-93e4-23a94af9eed4.png)
* there should not be any console errors related to the actions e.g. `cannot find property getSchemaBlocksValidationResults of undefined`


## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #
